_docs/12-component-controllers.md

```diff
@@ -33,4 +33,48 @@ But sometimes, you do need an object with a longer lifecycle. *Component control
 
 There is a only a one-way communication channel between the component and its component controller - you can only pass data off of a component to a component controller. A component has no reference its corresponding component controller. This is by design. 
```

Comment:
@ocrickard : Yup, but I didn't update this section since it was already there, and since we discourage it anyway. I think this PR should focus on the new `initWithComponent:` method you added, and we can figure out separately whether to update this bit about components accessing their controllers.